### DISCUSSION TITLE
Question regarding Dockerfile

### DISCUSSION THREAD

**Discussion by dhirensr** [ersilia-os/ersilia#276] [General] [answered]
Hi Ersilia team,

I have noticed the dockerfile in the repository is minimalistic and doesn't does much. I was wondering if that's supposed to do something, if yes I can take a look at extending it, if we want to achieve something with it.

**Comment by miquelduranfrigola** [comment]
Hi @dhirensr thanks for raising this issue! I agree with you, the current dockerfile should not be there. We have someone working on a Dockerfile and we've made some progress, although we are not quite there yet: https://github.com/ersilia-os/ersilia/blob/master/dockerfiles/installer/Dockerfile . Would you mind having a look at this more advanced dockerfile and let me know what you think?
Many thanks!

**Reply by dhirensr** [reply]
Hey @miquelduranfrigola ,
the dockerfile seems fine the one you linked, can be a bit improved. 
can I ask in general what you want to achieve with docker ? are you planning to use it for github actions testing or to make it launchable for users/developers?

**Comment by miquelduranfrigola** [comment, marked as answer]
Hello @dhirensr I am closing this issue now since we have tackled the problem with our colleagues. I should be able to update soon the repo soon.
